MovieApp/MovieApp/MoviesViewController.m

```diff
@@ -19,8 +26,17 @@ @implementation MoviesViewController
 - (void)viewDidLoad {
     
     [super viewDidLoad];
+    [self configureView];
+
+    downloader = [[MovieDBDownloader alloc] init];
+    [downloader configure];
```

Comment:
Configuration part should be moved to AppDelegate and initialized on app startup only once for all mappings.  Object manager is shared object manager and you should make this class utility class (with all class methods). Let me know if there is some reason you can't do this.